eng/tox/install_depend_packages.py

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env python
+
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+
+import argparse
+import os
+import sys
+from subprocess import check_call
+import logging
+from packaging.specifiers import SpecifierSet
+from pkg_resources import Requirement
+from pypi_tools.pypi import PyPIClient
+
+setup_parser_path = os.path.abspath(
+    os.path.join(os.path.abspath(__file__), "..", "..", "versioning")
+)
+sys.path.append(setup_parser_path)
+from setup_parser import get_install_requires
+
+DEV_REQ_FILE = "dev_requirements.txt"
+NEW_DEV_REQ_FILE = "new_dev_requirements.txt"
+PKGS_TXT_FILE = "packages.txt"
+EXCLUDED_PKGS = ['azure-common',]
```

Comment:
Why do we exclude this here, but still build the package? Probably just forgetting a situation you already introduced.